docs/bill_of_materials.md

```diff
@@ -50,19 +50,28 @@ You need a WiFi solution for each JetBot.  Select an option and get the parts sp
 
 #### Option 1 (default) - M2 Card + Antennas
 
+This is the good option for Jetson Nano (4GB) Developr Kit.<br>
```

Comment:
This option is a good choice if you're building JetBot with the original Jetson Nano 4GB.  It has been widely tested and used for JetBot.

!!! important

    Jetson Nano 2GB does not have a M.2 connector, so this option will not work with it.